ci/lint/pydoclint-baseline.txt

```diff
@@ -125,9 +125,6 @@ python/ray/_private/resource_isolation_config.py
     DOC103: Method `ResourceIsolationConfig.__init__`: Docstring arguments are different from function arguments. (Or could be other formatting issues: https://jsh9.github.io/pydoclint/violation_codes.html#notes-on-doc103 ). Arguments in the function signature but not in the docstring: [cgroup_path: Optional[str], enable_resource_isolation: bool, system_reserved_cpu: Optional[float], system_reserved_memory: Optional[int]].
     DOC201: Method `ResourceIsolationConfig._validate_and_get_system_reserved_cpu` does not have a return section in docstring
 --------------------
-python/ray/_private/resource_spec.py
-    DOC201: Method `ResourceSpec.resolve` does not have a return section in docstring
---------------------
```

Comment:
This file has been deleted on master. I'm not sure why lint is giving me grief on this PR but not on master